Makefile

```diff
@@ -64,13 +64,15 @@ INCLUDEOPTS=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
 	    | grep '^ /' | grep -v cc1plus | awk '{print "-I" $$1 " "}')
 PYINCLUDE=$(shell python -c \
 	  "import sysconfig;print(sysconfig.get_path('include'))")
```

Comment:
You need at least one comma to make a tuple: `type((23))` -> `int` and `type((23,))` -> `tuple`. So it isn't tuple unpacking but the fact that `(23)` is a noop. Legacy python you so cray cray :)

This means we don't need the print function import in the other case either. I think.

(spot the person waiting for a plane and too much time ...)